web/src/components/LoginLocked.test.js

```diff
@@ -25,7 +25,9 @@ describe('<LoginLocked />', () => {
     const { getByText } = renderUtils;
     expect(
       getByText((content, node) => {
-        const hasText = () => node.textContent === 'Contact CMS-EAPD@cms.hhs.gov for an account reset.';
+        const hasText = () =>
+          node.textContent ===
+          'Contact CMS-EAPD@cms.hhs.gov for an account reset.';
```

Comment:
You could potentially use less indentation, here.

`@testing-library` example: https://github.com/CMSgov/eAPD/pull/2838/files#diff-8316771e25fcd1cf4c293df2dac390434d329cb9abf93c8e9ab0caab732d3a3cR36-R40

enzyme example: https://github.com/CMSgov/eAPD/blob/master/web/src/components/ContinuePreviousButtons.test.js#L48-L63

Reply:
I didn't actually change that, I think that was a prettier change

Reply:
Right. I was just pointing out that the test could be restructured so that there is less indentation and so that it is easier to read. No change necessary, just an observation.